### PR TITLE
fix: better mock class for NotValidVar

### PR DESCRIPTION
**Pull request by lekaha**

### CODE REVIEW

File: ext/src/test/java/com/playone/mobile/common/NotValidValueTest.kt [hunk 1]
```diff
@@ -12,7 +13,7 @@ class MockClass {
 }
 
 data class MockDataClass(var _abc: Int = INVALID_INT) {
-    var abc by NotValidVar(_abc)
+    var abc by NotValidVar(if (_abc != INVALID_INT) _abc else INVALID_INT)
```
Comment (pokk) [inline ext/src/test/java/com/playone/mobile/common/NotValidValueTest.kt]: `_abc.takeIf{ INVALID_INT != it } ?: INVALID_INT`
will be better?!

Comment (pokk) [reply ext/src/test/java/com/playone/mobile/common/NotValidValueTest.kt]: Or we might make a function for checking the invalid value.

`_abc.takeIf(Int::isValid) ?: INVALID_INT` or something.

**Comment by lekaha** [comment]
@pokk Please check again

**Review by pokk — APPROVED** [review]
Looks good to me.

**Pull request merged**